e2etests/testdata/regression/elk_order/dagre/sketch.exp.svg

```diff
@@ -3,7 +3,7 @@
 id="d2-svg"
 style="background: white;"
 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
-width="1540" height="578" viewBox="-102 -102 1540 578"><style type="text/css">
+width="1170" height="518" viewBox="-102 -102 1170 518"><style type="text/css">
```

Comment:
i feel like there should've been some padding on this on the right, so the bounding box isn't touching the queue circle thing

Reply:
agree but this is due to elk and dagre having fixed container padding with their layouts

Reply:
#754